### ISSUE TITLE
Sort numbers numerically in the Table View

### ISSUE THREAD

**Issue by arcturus2** [constellation-app/constellation#1197]
### Prerequisites

- [X] Put an X between the brackets on this line if you have done all of
    the following:

    -   Running the latest version of Constellation

    -   Attached the ***Support Package*** via `Help` &gt;
        `Support Package`

    -   Checked the FAQs:
        https://github.com/constellation-app/constellation/wiki/FAQ

    -   Checked that your issue isn’t already filed:
        https://github.com/constellation-app/constellation/issues

    -   Checked that there is not already a module that provides the
        described functionality:
        https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

The Table view sorts columns with numbers alphanumerically instead of numerically. For example, instead of:

```
1
10
100
2
```

it should be:

```
1
2
10
100
```

### Steps to Reproduce

1. File -> New Graph
2. Experimental -> Build Graph -> Small world graph
3. Open the Table View
4. Switch to the nodes mode
5. Add the x attribute as a column
6. Sort the column

**Expected behaviour:** \[What you expect to happen\]
Column should be sorted numerically

**Actual behaviour:** \[What actually happens\]
Column is sorted as Strings instead

**Reproduces how often:** \[What percentage of the time does it reproduce?\]
100

**Comment by antares1470** [comment]
A quick dig into the code confirms that the table has been constructed such that all inputs are Strings (hence all columns including ones that would appear to be numeric are sorted lexicographically). A refactor of a number of parts of the table would be required then in order to be able to handle numerical values properly

**Comment by sol695510** [comment]
@antares1470 I first attempted to refactor the table columns and associated variables to the type of Object so that the columns themselves would infer the type, String or Integer, then sort accordingly.

That didn't seem to work so I tried implementing a custom comparator on the SortedList that populates the columns. It was either not being bound properly or being overridden at a later stage so that didn't affect the sorting either.

Speaking with @formalhaunt, a possible solution is to implement a comparator on the table columns as they are being created in Table.java.
